apps/web/frontend/components/combobox/combobox.tsx

```diff
@@ -8,7 +8,7 @@ interface ComboBoxProps<TOption extends { id: TIdType }, TIdType extends string>
   onChange: (id: TIdType | null) => void
   displayValue: (option: TOption) => string
   onBlur?: () => void
-  options: TOption[]
+  options: Readonly<TOption[]>
```

Comment:
```suggestion
  options: ReadonlyArray<TOption>
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html